src/main/kotlin/no/nav/omsorgspenger/soknadOverforeDager/SøknadOverføreDagerValidator.kt

```diff
@@ -0,0 +1,78 @@
+package no.nav.omsorgspenger.soknadOverforeDager
+
+import no.nav.helse.dusseldorf.ktor.core.ParameterType
+import no.nav.helse.dusseldorf.ktor.core.Throwblem
+import no.nav.helse.dusseldorf.ktor.core.ValidationProblemDetails
+import no.nav.helse.dusseldorf.ktor.core.Violation
+import no.nav.omsorgspenger.soknad.erGyldigNorskIdentifikator
+import no.nav.omsorgspenger.soknad.valider
+
+internal fun SøknadOverføreDager.valider() {
+    val violations: MutableSet<Violation> = mutableSetOf<Violation>()
+
+    if(!harSamfunnskritiskJobb){
```

Comment:
Skal det alltid være en valideringsfeil dersom harSamfunnskritiskJobb = false?
Det er det som sjekkes for her.

Reply:
Ja, kun dersom man har sagt ja til å ha samfunnskritisk jobb så skal man kunne overføre dager

Reply:
Ok, da er det riktig 👍